dodola/core.py

```diff
@@ -381,6 +381,11 @@ def standardize_gcm(ds, leapday_removal=True):
         if ds_cleaned.time.dtype == "int64":
             ds_cleaned["time"] = xr.decode_cf(ds_cleaned).time
         if cal == "360_day":
+
+            # Cannot have chunks in time dimension for 360 day calendar conversion so loading
+            # data into memory.
+            ds.load()
```

Comment:
Doesn't this need to be something like
```suggestion
            ds_cleaned.load()
```

Reply:
Ah.. thank you for catching this @brews !